src/main/java/school/faang/task_49272/Game.java

```diff
@@ -0,0 +1,35 @@
+package school.faang.task_49272;
+
+public class Game {
+    private int score = 0;
+    private int lives = 10;
+
+    private final Object scoreLock = new Object();
+    private final Object livesLock = new Object();
+
+
+    public synchronized boolean update(boolean earnedPoint, boolean isDed) {
+        synchronized (scoreLock) {
```

Comment:
Small optimization: if earnedPoint == false then there is no need to acquire the lock